app/_landing_pages/tools.yaml

```
@@ -0,0 +1,150 @@
+metadata:
+  title: "Kong Tools"
+  tags:
+    - deck
+    - kic
+    - terraform
+    - konnect-api
+    - admin-api
+content:
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h1
+              text: "Kong Tools"
+              align: center
+
+  - row:
+      - column:
+          - type: structured_text
+            config:
+              header:
+                text: "What are Kong tools?"
+              blocks:
+                - type: text
+                  text: Rate limiting controls the number of API requests a client can make within a specified time frame. 
+                    It's a critical function that helps maintain
+                    the performance and availability of web
+                    services and APIs by controlling the request traffic.
```

```suggestion
                  text: There are three types of tools availble for interacting with Kong Gateway
                    APIs: The Admin API and the control plane configuration API. 
                    SDKs: 
                    CLIs: CLI utilities like decK are available for configuring KGW and Konnect
```

Really rough idea of a way we could split this into being about the three tool types if we decide to go that route. Then we can split the cards at the bottom by those types.